specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/library.json

```diff
@@ -0,0 +1,479 @@
+          },
+          {
+            "$ref": "#/parameters/OperationId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok.",
+            "schema": {
+              "$ref": "#/definitions/LibraryResource"
+            }
+          },
+          "202": {
+            "description": "Accepted.",
+            "schema": {
+              "$ref": "#/definitions/OperationResult"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Azure Synapse service.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/libraries/{libraryName}": {
+      "delete": {
+        "tags": [
+          "Library"
+        ],
+        "operationId": "Library_Delete",
+        "x-ms-examples": {
+          "Delete Library": {
+            "$ref": "./examples/Library_Delete.json"
+          }
+        },
+        "description": "Delete Library",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/LibraryNameParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted."
+          },
+          "409": {
+            "description": "Conflict."
+          },
+          "default": {
+            "description": "An error response received from the Azure Synapse service.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "Library"
+        ],
+        "operationId": "Library_Get",
+        "description": "Get Library",
+        "x-ms-examples": {
+          "Get Library": {
+            "$ref": "./examples/Library_Get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/LibraryNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "#/definitions/LibraryResource"
+            }
+          },
+          "304": {
+            "description": "Not modified."
+          },
+          "default": {
+            "description": "An error response received from the Azure Synapse service.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "LibraryPayload": {
+      "description": "Library content",
+      "type": "object",
+      "properties": {
+        "content": {
+          "type": "string",
+          "description": "Stream of library content"
+        }
+      }
+    },
+    "OperationResult": {
+      "description": "Operation status for the operation",
```

Comment:
If this is Azure standard, then we don't need to define its scheme explicitly in this file